ert3/console/_console.py

```diff
@@ -205,6 +231,33 @@ def _clean(workspace: Path, args: Any) -> None:
     ert3.console.clean(workspace, args.experiment_names, args.all)
 
 
+def _service_check(args: Any) -> None:
+    if args.service_name == "storage":
+        for _ in range(args.timeout):
+            try:
+                info = get_info()
+                print(f"Ert storage found: {info['baseurl']}")
+                return
+            except RuntimeError:
+                time.sleep(1)
+                continue
+        raise SystemExit("ERROR: Ert storage not found!")
+
+
+def _service_start(args: Any) -> None:
+    if args.service_name == "storage":
+        os.execvp("ert", ["ert", "api"])
+
+
+def _service(args: Any) -> None:
+    assert args.sub_cmd == "service"
+
+    if args.service_cmd == "check":
+        _service_check(args)
+    elif args.service_cmd == "start":
+        _service_start(args)
+
```

Comment:
```python
else:
    Kaboom("Not implemented")
```

A list of `elif` statements that should cover the entire range should not let non-matching candidates fall through :)